yaksh/models.py

```diff
@@ -840,6 +840,13 @@ class QuestionPaper(models.Model):
 
     objects = QuestionPaperManager()
 
+    def get_question_bank(self):
```

Comment:
Just a suggestion, we already have a function to get all questions for a question paper.
You can try that instead of this extra function.